GameData/RealismOverhaul/TestFlight_Generic_Engines.cfg

```diff
@@ -73,7 +73,7 @@
 	@TESTFLIGHT:HAS[~explicitDataRate[?rue]]
 	{
 		@reliabilityDataRateMultiplier *= 640 // normalized to rate=4 at 160s burntime
-		@reliabilityDataRateMultiplier /= #$ratedBurnTime$
+		@reliabilityDataRateMultiplier /= #$ratedContinuousBurnTime$
```

Comment:
This is gonna explode because that value isn’t guaranteed to exist until line 303.